portfolio/src/main/java/com/google/sps/servlets/Sentence.java

```diff
@@ -0,0 +1,62 @@
+
+package com.google.sps;
+
+import java.util.Comparator;
+
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Entity;
+import com.google.cloud.language.v1.Sentiment;
+import com.google.cloud.language.v1.Document;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+import java.util.List;
+import java.io.IOException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.Query;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+
+public final class Sentence {
+
+  private final String text;
+  private final float score;
+
+  public Sentence(String text){
+    this.text = text;
+    this.score = getScore(text);
+
+  }
+
+  public float getScore(String text){
+
+    try {
+    Document doc =
+        Document.newBuilder().setContent(text).setType(Document.Type.PLAIN_TEXT).build();
+    LanguageServiceClient languageService = LanguageServiceClient.create();
+    Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+    float sentScore = sentiment.getScore();
+    languageService.close();
+
+    return sentScore;
+    } catch(IOException e) {
```

Comment:
Why would this exception be hit?  At least add a comment explaining this, and justifying your fallback of -1.

Reply:
When I leave the exception out it says something like "uncaught java.io exception", I think this error isn't encountered when I use the same code on the servlets cause the function throws the io exception.

I relied on -1 cause ~at least I think~ you have to return a float? I relied on -1 cause the sentiment scores are usually never actually equal to 1 or -1 so if I guess if the sentence average was -1 I would know? But Idk if there's something better to put here.